sormas-ui/src/main/java/de/symeda/sormas/ui/caze/AbstractCaseView.java

```diff
@@ -168,6 +168,10 @@ public void refreshMenu(SubMenu menu, Label infoLabel, Label infoLabelSub, Strin
 				&& !FacadeProvider.getFeatureConfigurationFacade().isFeatureDisabled(FeatureType.CLINICAL_MANAGEMENT)) {
 				menu.addView(TherapyView.VIEW_NAME, I18nProperties.getPrefixCaption(CaseDataDto.I18N_PREFIX, CaseDataDto.THERAPY), params);
 			}
+		}
+		menu.addView(CaseVisitsView.VIEW_NAME, I18nProperties.getPrefixCaption(CaseDataDto.I18N_PREFIX, CaseDataDto.VISITS), params);
+
+		if (!hasOutbreak || !caze.getDisease().usesSimpleViewForOutbreaks() || viewConfiguration.getViewMode() != ViewMode.SIMPLE) {
```

Comment:
Remove this line

Reply:
Not sur to understand why?

To insert the `CaseVisitsView` tab next to `ClinicalCourseView` I've split a global `if (!hasOutbreak || !caze.getDisease().usesSimpleViewForOutbreaks() || viewConfiguration.getViewMode() != ViewMode.SIMPLE)`

It was: 
```java
if (!hasOutbreak || !caze.getDisease().usesSimpleViewForOutbreaks() || viewConfiguration.getViewMode() != ViewMode.SIMPLE) {
  …
  menu.addView(TherapyView.VIEW_NAME, …)
  menu.addView(ClinicalCourseView.VIEW_NAME, …)
  …
}
```

It became:
```java
if (!hasOutbreak || !caze.getDisease().usesSimpleViewForOutbreaks() || viewConfiguration.getViewMode() != ViewMode.SIMPLE) {
  …
  menu.addView(TherapyView.VIEW_NAME, …)
}
menu.addView(CaseVisitsView.VIEW_NAME, …)
if (!hasOutbreak || !caze.getDisease().usesSimpleViewForOutbreaks() || viewConfiguration.getViewMode() != ViewMode.SIMPLE) {
  menu.addView(ClinicalCourseView.VIEW_NAME, …)
  …
}
```

Reply:
You're right! Would be better to move the condition to a local variable and use it in both occasions then, though.